src/Idle.DataAccess/Fields/LanguageBase.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Idle.DataAccess.Buffs;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Idle.DataAccess.Fields
+{
+    /// <summary>
+    /// Abstract base class for all languages
+    /// </summary>
+    public abstract class LanguageBase : ModelBase, IDescriptive, IBuffing, IProgress, ICost
+    {
+        // Descriptive Properties
+        /// <summary>
+        /// Name of the language
+        /// </summary>
+        public abstract string Name { get; }
+        /// <summary>
+        /// Description of the language
+        /// </summary>
+        public abstract string Description { get; }
+        /// <summary>
+        /// Difficulty of the language
+        /// </summary>
+        public abstract ModelBase.Difficulty Difficulty { get; }
+
+
+
+        // Buff Properties
+        private float _xpMult = 1f;
+        /// <summary>
+        /// Base XP Multiplier of this language, the starter value may vary depending on the language
+        /// </summary>
+        public float XPMult { get => _xpMult; set => _xpMult = value; }
```

Comment:
I like the way us wrote this in one line. Looks good when the line is not too long.
But I would use auto-properties where possible.
so instead what do you think of writing
```cs
public float XPMult {get;set;} = 1f;
```
this would also eliminate the fields.

Reply:
Oh wow, I was unaware you could use auto-properties like that. Thank you.